tests/ssg_test_suite/oscap.py

```diff
@@ -273,6 +274,11 @@ def _generate_report_file(self):
         ])
         self._filenames_to_clean_afterwards.add(self.report_path)
 
+    def _wait_for_continue(self):
+        """ In case user requests to leave machine in failed state for hands
+        on debugging, ask for keypress to continue."""
+        raw_input("Paused for manual debugging. Continue by pressing return.")
```

Comment:
NACK, `NameError: name 'raw_input' is not defined`

Reply:
In `ssg` module, we have a wrapper for this, `ssg.shims.input_func`: https://github.com/OpenSCAP/scap-security-guide/blob/master/ssg/shims.py#L38-L42

Reply:
Thanks @cipherboy